src/main/scala/machines/Internal.scala

```diff
@@ -3,4 +3,31 @@ package machines
 import regex._
 import dfa._
 
-// TODO: Add your code below
+// Part 1
+given Conversion[Char, RegularLanguage] = n => Character(n)
+
+// Part 2
+given Conversion[String, RegularLanguage] = n => n.foldRight(Epsilon)((a: Char, b: RegularLanguage) => Concat(Character(a), b))
```

Comment:
I honestly like the way you wrote this one better than the way I wrote it. I got stuck on this one for a few minutes but ultimately ended up using map and reduce